packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -339,49 +322,48 @@ const Select = <
         event.stopPropagation();
 
       return (
-        <ChipsInnerContainer
-          isInteractive={isInteractive}
-          odysseyDesignTokens={odysseyDesignTokens}
-        >
-          {selection.length <= 0 && (
-            <ChipsSpacer odysseyDesignTokens={odysseyDesignTokens} />
-          )}
-          {selection.map(
-            (item: string) =>
-              item?.length > 0 && (
-                <MuiChip
-                  key={item}
-                  label={
-                    <>
-                      {item}
-                      {!isInteractive &&
-                        controlledStateRef.current === CONTROLLED &&
-                        hasMultipleChoices && (
-                          <NonInteractiveIcon
-                            odysseyDesignTokens={odysseyDesignTokens}
-                          />
-                        )}
-                    </>
-                  }
-                  tabIndex={-1}
-                  onDelete={
-                    isInteractive && controlledStateRef.current === CONTROLLED
-                      ? () => removeSelection(item)
-                      : undefined
-                  }
-                  deleteIcon={
-                    <CloseCircleFilledIcon
-                      sx={{ pointerEvents: "auto" }}
-                      // We need to stop event propagation on mouse down to prevent the deletion
-                      // from being blocked by the Select list opening, and also ensure that
-                      // the pointerEvent is registered even when the parent's are not
-                      onMouseDown={stopPropagation}
-                    />
-                  }
-                />
-              ),
-          )}
-        </ChipsInnerContainer>
+        Array.isArray(internalSelectedValues) && (
+          <ChipsInnerContainer
+            isInteractive={isInteractive}
+            odysseyDesignTokens={odysseyDesignTokens}
+          >
+            {internalSelectedValues.map(
+              (item: string) =>
+                item?.length > 0 && (
```

Comment:
@jordankoschei-okta What are we looking to accomplish with the length check on `item` here? Can item be `null` or empty string or something? Could we just do something like this?
```js
item && (
```

Reply:
For some reason, due to how MUI handles these (?), an empty internalSelectedValues will sometimes have a single empty item. This length check is how we confirm that this is a real item, not a blank.

Reply:
Not sure what you mean by 'will sometimes have a single empty item'. Is that just an empty string? What would the difference between an empty item and a 'blank' be?